portfolio/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Rachel Greenwood Portfolio</title>
+        <meta charset="utf-8">
+        <link rel="stylesheet" type="text/css" href="index.css" />
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=DotGothic16">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Silkscreen">
+    </head>
+    <body>
+        <nav><header><a href="#about">About</a><a href="#projects">Projects</a><a href="#contact">Contact</a></header><p id="now-playing">Now Playing: Selino - Tim Kahn</p><audio style="display:none" autoplay loop><source src="698343__timkahn__selino.wav"></audio></nav>
+        <!--"15 Solar Sailer.wav-->
+        <!--"698343__timkahn__selino.wav"-->
+        <main>
+            <div class="header">
+                <h1>Hi, I'm Rachel Greenwood</h1>
+                <h2>A Software Engineering Student at Techtonica</h2>
+            </div>
+            <div id="about">
+                <div class="first-impression">
+                    <img class="me" src="red.jpeg" width="50px">
+                    <p>I'm on week 3 of learning the PERN stack in Techtonica's 6 month program to help women and non-binary adults break into the tech industry</p>
+                </div>
+                <div class="columns">
+                    <p class="section1">I have a background as a Choir Director. My musical training lends to my software development capabilities:</p>
```

Comment:
Column layout leaves a bit to be desired when resized. Not sure if that's a problem at this point, but it's worth noting.
![Screen Shot 2023-08-04 at 8 23 15 PM](https://github.com/RachelGreenwood/TechtonicaAssignments/assets/892249/35858b50-650d-459e-8a40-eb3da236016a)